src/components/NcTimezonePicker/NcTimezonePicker.vue

```diff
@@ -44,13 +44,8 @@ export default {
 	<NcMultiselect :value="selectedTimezone"
 		:options="options"
 		:multiple="false"
-		:group-select="false"
 		:placeholder="placeholder"
-		group-values="regions"
-		group-label="continent"
```

Comment:
Does this mean that we lose grouping values ?

Reply:
Will revert some breaking changes discussed with @PVince81